include/nng/supplemental/nanolib/conf.h

```diff
@@ -248,6 +248,7 @@ struct conf_bridge_node {
 	uint8_t      proto_ver;
 	uint16_t     port;
 	uint16_t     keepalive;
+	uint16_t     backoff_max;
```

Comment:
Could uint16_t be too small and nni_duration be a better choice?

Reply:
a solid concern. but nobody wants a loooooooog sleep backoff